features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/ResourcesFeature.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal
+
+import android.content.Context
+import com.datadog.android.api.feature.FeatureSdkCore
+import com.datadog.android.api.feature.StorageBackedFeature
+import com.datadog.android.api.net.RequestFactory
+import com.datadog.android.api.storage.FeatureStorageConfiguration
+import com.datadog.android.sessionreplay.internal.domain.ResourceRequestFactory
+import com.datadog.android.sessionreplay.internal.storage.NoOpResourceWriter
+import com.datadog.android.sessionreplay.internal.storage.SessionReplayResourceWriter
+import java.util.concurrent.atomic.AtomicBoolean
+
+/**
+ * Session Replay feature class, which needs to be registered with Datadog SDK instance.
+ */
+internal class ResourcesFeature(
+    private val sdkCore: FeatureSdkCore
+) : StorageBackedFeature {
+
+    internal var dataWriter: ResourceWriter = NoOpResourceWriter()
+    internal val initialized = AtomicBoolean(false)
+
+    // region Feature
+
+    override val name: String = SESSION_REPLAY_RESOURCES_FEATURE_NAME
+
+    override val requestFactory: RequestFactory = ResourceRequestFactory(
+        customEndpointUrl = null,
```

Comment:
are we going to support custom endpoint? otherwise there is an inconsistency with `Session Replay` feature which supports it (if it is set there, then wireframes will be uploaded to other host than resources)

Reply:
I've changed to pass in the custom endpoint from the session replay feature